platform_versioned_docs/version-24.1/data_studios/index.mdx

```diff
@@ -0,0 +1,112 @@
+- [Data Explorer](../data/data-explorer.mdx) is enabled.
+
+:::note
+Currently, Data Studios only supports AWS Batch compute environments that **do not** have Fargate enabled.
+:::
+
+## Container image templates
+
+Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the session container is built. Users have permission to install additional packages as needed.
+
+In addition, you can create your own container image templates. For more information, see [Custom environments][custom-envs].
+
+**JupyterLab 4.2.5**
+
+The default user is the `root` account. The following [conda-forge](https://conda-forge.org/) packages are available by default:
+
+- `python=3.13.0`
+- `pip=24.2`
+- `jedi-language-server=0.41.4`
+- `jupyterlab=4.2.5`
+- `jupyter-collaboration=1.2.0`
+- `jupyterlab-git=0.50.1`
+- `jupytext=1.16.4`
+- `jupyter-dash=0.4.2`
+- `ipywidgets=7.8.4`
+- `pandas[all]=2.2.3`
+- `scikit-learn=1.5.2`
+- `statsmodels=0.14.4`
+- `itables=2.2.2`
+- `seaborn[stats]=0.13.2`
+- `altair=5.4.1`
+- `plotly=5.24.1`
+- `r-ggplot2=3.5.1`
+- `nb_black=1.0.7`
+- `qgrid=1.3.1`
+
+To install additional Python packages, execute `!pip install <packagename>` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
+
+**RStudio Server 4.4.1**
+
+The default user is the `root` account. To install R packages, execute `install.packages("<packagename>")` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
+
+**Visual Studio Code 1.93.1**
+
+[Visual Studio Code][vsc] is an integrated development environment (IDE) that supports many programming languages. The default user is the `root` account. To install extensions, select **Extensions**. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
+
+**Xpra 6.2.0**
+
+[Xpra](https://github.com/Xpra-org/xpra), known as _screen for X_, allows you to run X11 programs by giving you remote access to individual graphical applications. The container template image also installs NVIDIA Linux x64 (AMD64/EM64T) drivers for Ubuntu 22.04 for running GPU-enabled applications. To use these GPU drivers, make sure that your compute environment specifies GPU instance families.
+
+The default user is the `root` account. The image is based on `ubuntu:jammy`. Additional system-level packages can be installed in a terminal window using `apt install <package_name>`.
+
+## Session statuses
+
+Data studios have the following possible statuses:
+
+- **building**: When a custom environment is building the template image for a new data studio session. The [Wave] service performs the build action. For more information on this status, see [Inspect custom container template build status][build-status].
```

Comment:
I know we use code format in another list of statuses, so would be good to be consistent. This is a UI component (hence being bolded) so maybe the other list can be bolded to match it.